webhooks/virtualmachine/mutation/virtualmachine_mutator_intg_test.go

```diff
@@ -113,6 +114,55 @@ func intgTestsMutating() {
 		})
 	})
 
+	Context("SetDefaultBiosUUID", func() {
+		When("Creating VirtualMachine", func() {
+			When("When VM BiosUUID is empty", func() {
+				BeforeEach(func() {
+					ctx.vm.Spec.BiosUUID = ""
+				})
+
+				It("Should set BiosUUID to PoweredOn", func() {
```

Comment:
typo: 'to PoweredOn'

Reply:
Whoops, fixed. I'd copy-n-pasted `It` block from another test